### ISSUE TITLE
NoMultipleSpaces triggered for annotation

### ISSUE THREAD

**Issue by blundell** [detekt/detekt#6420]
I have this in my code.

Screenshot:

<img width="697" alt="image" src="https://github.com/detekt/detekt/assets/655860/41160272-1b82-4f86-acbf-1f159946af11">

code:

```
@Suppress(
    "UnusedPrivateMember", // a comment explaining why this suppression
    "ComplexMethod",  // a different comment explaining why this suppression
)
```

But detekt is triggered:

> NoMultipleSpaces - [Unnecessary long whitespace] at com.blundell.Foo.kt:106
> NoMultipleSpaces - [Unnecessary long whitespace] at com.blundell.Foo.kt:107

Config for the rule:

```
  NoMultipleSpaces:
    active: true
    excludes: "**/build/**"
    autoCorrect: true
```

## Expected Behavior of the rule

1) I think the formatting is perfectly fine in the above image, my (kotlin defaults) auto-formatter actively changes it to this. (i.e. the rule shouldn't trigger)

2) I was hoping that adding `ignoreAnnotation: true` to the config would allow `NoMultipleSpaces` to ignore annotations, but it does not?

## Context

I would like to keep my suppressions readable, with a contextual comment next to each suppression.

**Comment by blundell** [comment]
Lol, ignore me! It's the spaces after the comma before the comment that are triggering it 😆 

Fixed:

```
@Suppress(
    "UnusedPrivateMember", // a comment explaining why this suppression
    "ComplexMethod", // a different comment explaining why this suppression
)
```
